terraform/layers/base/providers.tf

```diff
@@ -9,6 +9,9 @@ terraform {
 }
 
 provider "aws" {
+
+  region = "eu-west-2"
```

Comment:
Mentioned on previous PR, this would be better as a variable called 'region'

Reply:
Have moved to variables.tf @PAYNEA03